blocks/a11y-testing-block/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@wpmedia/a11y-testing-block",
-  "version": "5.10.2",
+  "version": "5.10.3-hotfix.0",
```

Comment:
interesting, it looks like this versioning updated the version of all the packages @matthewroach . what command did you use for this?